src/sentry/static/sentry/app/views/usageStats/index.tsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router';
+import styled from '@emotion/styled';
+import capitalize from 'lodash/capitalize';
+import moment from 'moment';
+
+import {Client} from 'app/api';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import Card from 'app/components/card';
+import {HeaderTitle} from 'app/components/charts/styles';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import PageHeading from 'app/components/pageHeading';
+import {Panel, PanelBody} from 'app/components/panels';
+import QuestionTooltip from 'app/components/questionTooltip';
+import {t, tct} from 'app/locale';
+import {PageContent, PageHeader} from 'app/styles/organization';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {DataCategory, Organization} from 'app/types';
+
+import Chart from './chart';
+import {OrganizationUsageStats, ProjectUsageStats} from './types';
+import {formatUsageWithUnits} from './utils';
+import withUsageStats from './withUsageStats';
+
+type Props = {
+  api: Client;
+  organization: Organization;
+
+  orgStats?: OrganizationUsageStats;
+  orgStatsLoading: boolean;
+  orgStatsError?: Error;
+
+  projectStats?: ProjectUsageStats[];
+  projectStatsLoading: boolean;
+  projectStatsError?: Error;
+} & RouteComponentProps<{orgId: string}, {}>;
+
+type State = {
+  dataCategory: DataCategory;
+};
+
+class OrganizationStats extends React.Component<Props, State> {
+  state: State = {
+    dataCategory: DataCategory.ERRORS,
+  };
+
+  setSelectedDataCategory = (dataCategory: DataCategory) => {
+    this.setState({dataCategory});
+  };
+
+  get selectedDataCategory() {
+    return capitalize(this.state.dataCategory);
+  }
+
+  /**
+   * Ignore this method. API response is being changed so this will be amended
+   * in a few days.
+   */
+  get formattedOrgStats(): {
+    stats: any[];
+    total: string;
+    accepted: string;
+    dropped: string;
+    filtered: string;
+    overQuota: string;
+  } {
```

Comment:
Ignore this method. API response is being changed so this will be amended in a few days.